targets/pna_nic/pnic_CLI.py

```diff
@@ -48,7 +48,28 @@ def do_get_time_since_epoch(self, line):
         print(self.pnic_client.get_time_since_epoch_us())
 
 def load_json_pna(json):
-    pass
+    def get_json_key(key):
+        return json.get(key, [])
+
+    for j_meter in get_json_key("extern_instances"):
```

Comment:
I would remove the changes to this file from this PR. I am not sure why we are only adding support for meters, but if that's what we need, I would still do it in a follow-up PR.

Reply:
In this PR, No.